### PR TITLE
Improve and consolidate key Cloud content

### PR DESCRIPTION
**Pull request by seaxwi**
**Changes proposed in this pull request**
Consolidate content into new and updated articles.

* Renames [Add an Arduino device to IoT Cloud](https://support.arduino.cc/hc/en-us/articles/360016495559-Add-an-Arduino-device-to-IoT-Cloud)
* Renames [If your device is not connecting to IoT Cloud](https://support.arduino.cc/hc/en-us/articles/360019355679-If-your-device-is-not-connecting-to-IoT-Cloud) 
* Removes a bunch of other articles

**Internal key(s):** HC-991, HC-994, HC-995

### CODE REVIEW

**Review by Renat0Ribeir0 — CHANGES_REQUESTED** [review]
Good job with re-structuring the sections 👍 Added some suggestions

File: content/Arduino Cloud/General Cloud/Check-if-the-Arduino-Create-Agent-is-installed-and-running.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+---
+title: "Check if the Arduino Create Agent is installed and running"
+---
+
+Learn how to check if the **Arduino Create Agent** is installed and running on your computer.
```
[the remark this answers is not in the document]
Comment (seaxwi) [reply content/Arduino Cloud/General Cloud/Check-if-the-Arduino-Create-Agent-is-installed-and-running.md:5]: d6bc1d3

File: content/Arduino Cloud/Arduino IoT Cloud/Add-and-connect-a-device-to-IoT-Cloud.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
+---
+title: "Add and connect a device to IoT Cloud"
+---
+
+> Also see [Getting Started With the Arduino IoT Cloud](https://docs.arduino.cc/cloud/iot-cloud/tutorials/iot-cloud-getting-started) in Arduino Docs.
+
+Connecting a new device to Arduino IoT Cloud consists of three steps:
+
+<!--
```
[the remark this answers is not in the document]
Comment (seaxwi) [reply content/Arduino Cloud/Arduino IoT Cloud/Add-and-connect-a-device-to-IoT-Cloud.md]: d6bc1d3

File: content/Arduino Cloud/Arduino IoT Cloud/Add-and-connect-a-device-to-IoT-Cloud.md [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
+### Step 2: Create and configure a Thing
+
+<!-- The network parameters will be written to the `thingProperties.h` and `Secret` sketch files. But a sketch is only generated when a variable is added, so we'll do that first. -->
+
+1. Open the [Things tab](https://create.arduino.cc/iot/things).
+
+2. Click **Create Thing**.
+
+   ![](img/iot-thing-create.png)
```
[the remark this answers is not in the document]
Comment (seaxwi) [reply content/Arduino Cloud/Arduino IoT Cloud/Add-and-connect-a-device-to-IoT-Cloud.md]: d6bc1d3

File: content/Arduino Cloud/Arduino IoT Cloud/Add-and-connect-a-device-to-IoT-Cloud.md [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+
+3. Configure **Variables**, **Device**, and **Network** for the new Thing:
+
+   ![](img/thing-setup-before.png)
```
[the remark this answers is not in the document]
Comment (seaxwi) [reply content/Arduino Cloud/Arduino IoT Cloud/Add-and-connect-a-device-to-IoT-Cloud.md]: d6bc1d3

File: content/Arduino Cloud/Arduino IoT Cloud/Add-and-connect-a-device-to-IoT-Cloud.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+
+2. Click the **Upload button** and wait for the process to complete.
+
+   ![](img/iot-sketch-upload.png)
+
+3. When the upload is finished, go back to the Thing setup tab.
+
+4. After a short while, your device should come online. You may have to refresh the page.
+
+   ![](img/iot-device-online.png)
```
[the remark this answers is not in the document]
Comment (seaxwi) [reply content/Arduino Cloud/Arduino IoT Cloud/Add-and-connect-a-device-to-IoT-Cloud.md]: d6bc1d3

File: content/Arduino Cloud/Arduino IoT Cloud/Add-and-connect-a-device-to-IoT-Cloud.md [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+
+   > **Note:** Device status is not supported by the LoRa protocol, see [About device status in IoT Cloud](https://support.arduino.cc/hc/en-us/articles/4407169649682-About-device-status-in-IoT-Cloud).
+
+---
+
+## Troubleshooting
+
+*
```
[the remark this answers is not in the document]
Comment (seaxwi) [reply content/Arduino Cloud/Arduino IoT Cloud/Add-and-connect-a-device-to-IoT-Cloud.md]: d6bc1d3

File: content/Arduino Cloud/General Cloud/Check-if-the-Arduino-Create-Agent-is-installed-and-running.md [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+
+---
+
+1. _Make sure the agent is not paused._ Look for the ![Arduino Create Agent icon](img/create-agent-logo-mac.png) icon in the top-right of the menu bar (macOS and Linux) or bottom-right of the taskbar (Windows), usually within the system tray:
+
+   ![](img/create-agent-system-tray.png)
```
[the remark this answers is not in the document]
Comment (seaxwi) [reply content/Arduino Cloud/General Cloud/Check-if-the-Arduino-Create-Agent-is-installed-and-running.md]: d6bc1d3

File: content/Arduino Cloud/Arduino IoT Cloud/Add-and-connect-a-device-to-IoT-Cloud.md [hunk 8]
```diff
@@ -0,0 +1,96 @@
...
+   2. Click the **Network configure** button and enter the network credentials.
+
+      * _Wi-Fi:_ Enter your Wi-Fi name (SSID) and password (encryption key).
+      * _Arduino SIM_: [Use these credentials](https://support.arduino.cc/hc/en-us/articles/360013825159-What-are-the-credentials-for-the-Arduino-SIM-card).
+      _Arduino with LoRa<sup>®</sup>:_ The credentials are added automatically.
+      * _ESP32 and ESP8266_: Enter the secret key that was provided when adding the device.
+
+   3. Click **Add variable** and add one variable with any settings.
+
+   ![A Thing with an asociated device, network configuration, and added variable.](img/thing-setup-after.png)
```
Comment (per1234) [inline content/Arduino Cloud/Arduino IoT Cloud/Add-and-connect-a-device-to-IoT-Cloud.md:70]:
```suggestion
   2. Click **Add variable** and add one variable with any settings.

   ![A Thing with an asociated device, network configuration, and added variable.](img/thing-setup-after.png)

   3. Click the **Network configure** button and enter the network credentials.

      * _Wi-Fi:_ Enter your Wi-Fi name (SSID) and password (encryption key).
      * _Arduino SIM_: [Use these credentials](https://support.arduino.cc/hc/en-us/articles/360013825159-What-are-the-credentials-for-the-Arduino-SIM-card).
      _Arduino with LoRa<sup>®</sup>:_ The credentials are added automatically.
      * _ESP32 and ESP8266_: Enter the secret key that was provided when adding the device.
```

At the current time, the "**Network configure** button" is only enabled after adding a variable to the Thing. So they must be done in this order.

File: content/Arduino Cloud/Arduino IoT Cloud/Add-and-connect-a-device-to-IoT-Cloud.md [hunk 9]
```diff
@@ -0,0 +1,96 @@
...
+
+4. Your sketch will now be generated and can be viewed in the Sketch tab.
+
+<a id="upload-sketch"></a>
+
+### Step 3: Upload the Thing sketch
+
+1. With the Thing open, open the **Sketch** tab.
+
+2. Click the **Upload button** and wait for the process to complete.
```
Comment (per1234) [inline content/Arduino Cloud/Arduino IoT Cloud/Add-and-connect-a-device-to-IoT-Cloud.md:80]: Maybe a step should be added here about installing or starting Arduino Create Agent if it is not already running?

**Review by per1234 — CHANGES_REQUESTED** [review]

**Review by Renat0Ribeir0 — APPROVED** [review]

File: content/Arduino Cloud/Arduino IoT Cloud/Add-and-connect-a-device-to-IoT-Cloud.md (hunk 9 above)
Comment (seaxwi) [reply content/Arduino Cloud/Arduino IoT Cloud/Add-and-connect-a-device-to-IoT-Cloud.md:80]: Applies to "Add device" as well, added to introduction in 8dd0e25

**Pull request merged**